R/simulateHubbellRates.R

```diff
@@ -0,0 +1,163 @@
+#' Hubbell's neutral model simulation applied to time series
+#'
+#' Neutral species abundances simulation according to the Hubbell model. This
+#' model shows that losses in society can be replaced either by the birth of
+#' individuals or by immigration depending on their probabilities.
+#' The specific time between the events of birth or migration is calculated and
+#' time effect is considered to determine the next event.
+#'
+#' @param community_initial numeric value a vector of integers,
+#' containing species counts greater or equal to zero.
+#' @param migration_p numeric immigration possibility. It defines the
+#' probability of migration when replacement is needed in the community.
+#' The value can be between 0 and 1. The sum of the probability of
+#' migration and the probability birth must be 1.
+#' @param metacommunity_p numeric value the probability of a species being
+#' found in the metacommunity.
+#' @param k_events integer number of steps performed at a time point.
+#' It can be equal or more than 1. Bigger k_events increases speed while
+#' decreasing precision.
+#' @param growth_rates numeric rate of change in community size.
+#' @param norm logical scalar choosing whether the time series should be
+#' returned with the abundances as proportions (\code{norm = TRUE}) or
+#' the raw counts (default: \code{norm = FALSE})
+#' @param t.end numeric value of simulation end time
```

Comment:
be consistent, not both underscore and point, like `k_events` and `t.end`. Could this change to `t_end`? Similar changes also elsewhere.